src/util/instances.tsx

```diff
@@ -63,3 +63,7 @@ export const truncateInstanceName = (
 
   return name + suffix;
 };
+
+export const sanitizeInstanceName = (name: string): string => {
+  return name.replace(/[^A-Za-z0-9-]/g, "");
```

Comment:
```suggestion
  return name.replace(/[^A-Za-z0-9-]/g, "-");
```
Removing spaces can produce hard to read names, maybe better use a dash?